internal/pkg/v2/infrastructure/test/response.go

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright 2020 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+package test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// Unmarshal is a factory method that returns a structure.
```

Comment:
It does.  I realize it isn't immediately apparent.  You pass in `t`and `content` and a reference to a `value` and we unmarshal the `content` into the `value`.  Because `value` is passed in as a reference, when the function returns it now contains (assuming there wasn't an error) the structure unmarshalled from `content`.

Reply:
I will update the function's docblock comment.